src/validations/validator.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { ValidationFailedError } = require('@uphold/http-errors');
+const asserts = require('validator.js-asserts');
+const validator = require('@uphold/validator.js');
```

Comment:
This is currently in `devDependencies` (the above as well). Move this file to `test/utils` or something like that, to make it clearer that this is for tests only and would fail if used in public code?

Reply:
Yes, will move to `test/utils`.